grounding/src/main/java/com/sap/ai/sdk/grounding/GroundingClient.java

```diff
@@ -0,0 +1,89 @@
+package com.sap.ai.sdk.grounding;
+
+import com.google.common.annotations.Beta;
+import com.sap.ai.sdk.core.AiCoreService;
+import com.sap.ai.sdk.grounding.api.PipelinesApi;
+import com.sap.ai.sdk.grounding.api.RetrievalApi;
+import com.sap.ai.sdk.grounding.api.VectorApi;
+import javax.annotation.Nonnull;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Service class for the Grounding APIs.
+ *
+ * @since 1.3.0
+ */
+@Beta
```

Comment:
Any particular reason to make this beta? I guess most methods return Beta stuff anyway, so removing this essentially only stabilizes `create` 😄